chain/network/src/peer/peer_actor.rs

```diff
@@ -665,6 +665,10 @@ impl PeerActor {
                     },
                     Err(err) => {
                         tracing::info!(target: "network", "{:?}: Connection with {:?} rejected by PeerManager: {:?}", act.my_node_id(),conn.peer_info.id,err);
+                        act.network_state.config.event_sink.push(Event::ConnectionClosed(ConnectionClosedEvent{
+                            stream_id: act.stream_id,
+                            reason: ClosingReason::RejectedByPeerManager(err),
+                        }));
                         act.stop(ctx,ClosingReason::RejectedByPeerManager(err));
```

Comment:
the event is already reported here: https://cs.github.com/near/nearcore/blob/fe6c4b9c395fed6b39a25081fb1133dcf453ad29/chain/network/src/peer/peer_actor.rs?q=ConnectionClosed#L1197

aren't you duplicating the event then?